infra.bs

```diff
@@ -280,6 +280,81 @@ quotes and monospace font.
 <a>ASCII lowercase</a> of <var>B</var>.
 <!-- TODO: define string equals? -->
 
+To <dfn data-export="">collect a sequence of characters</dfn> means that the following algorithm
```

Comment:
I think it should be code points. That is what HTML says it means, at least: https://html.spec.whatwg.org/#character